Source/EasyNetQ/PullingConsumer.cs

```diff
@@ -334,11 +334,11 @@ public async Task<PullResult> PullAsync(CancellationToken cancellationToken = de
                     queue.Name
                 ),
                 new MessageProperties(basicGetResult.BasicProperties),
-                basicGetResult.Body.ToArray()
+                basicGetResult.Body
             );
             var interceptedMessage = interceptor.OnConsume(message);
             return PullResult.Available(
-                messagesCount, interceptedMessage.ReceivedInfo, interceptedMessage.Properties, interceptedMessage.Body
+                messagesCount, interceptedMessage.ReceivedInfo, interceptedMessage.Properties, interceptedMessage.Body.ToArray()
```

Comment:
Why not change `PullResult`'s property to ROM from array?

Reply:
Done